Dockerfile-cli.template

```
@@ -1,5 +1,6 @@
 {{ include "shared" -}}
-FROM alpine:3.18
+{{ include "template-helper-functions" }}
```

Need the `-` to tell it to slurp up the whitespace (so that it doesn't generate a blank line in the final Dockerfiles). It could be combined with the other `include`, but is fine separate as well.

```suggestion
{{ include "template-helper-functions" -}}
```